roles/init_dbserver/vars/PG_RedHat.yml

```diff
@@ -1,5 +1,6 @@
 ---
 pg_wal: ""
+pg_local_wal_archive_dir: ""
```

Comment:
You don't need to redefine this variable here. Initializing it only once in `defaults/main.yml` is sufficient.